distributed/shuffle/multi_file.py

```diff
@@ -160,7 +170,7 @@ async def communicate(self):
 
                     await self.queue.get()
 
-                id = max(self.sizes, key=self.sizes.get)
+                id = max(self.sizes, key=self.sizes.get)  # type: ignore
```

Comment:
```suggestion
                id = max(self.sizes, key=self.sizes.__getitem__)
```

As before, raises `ValueError` if `self.sizes` is empty.